### DISCUSSION TITLE
Expand Query Results pane to full width while keeping Primary Sidebar expanded above it?

### DISCUSSION THREAD

**Discussion by gregclerk** [microsoft/azuredatastudio#23555] [Q&A]
Is there a way to have the Query Results pane take up the entire width of the screen even while the Primary Sidebar is expanded above it?  I've tried to see if there was a way to shrink the vertical height of the Primary Sidebar but couldn't find it.

Ideally, the Query Results pane (when active) would just cover over the Primary Sidebar and vice versa.

Thanks

**Comment by erinstellato-ms** [comment]
@gregclerk The only options for the sidebar are to display on the left or side (via the setting Workbench > Side Bar: Location).

You can collapse the Sidebar (e.g. if you have the Connections sidebar open, click on the Connection icon in the Activity bar to close it). But there is no ability to re-size the Sidebar.

Does that help?

**Reply by gregclerk** [reply]
It answers my question but I wish there was a way to have the query results overlap any sidebar that is open.  The ability to resize the sidebar would be nice too for those that have monitors in portrait mode because the dialogs display at the bottom of the sidebar instead of right up underneath the other elements.  First world problems, I suppose.

**Reply by erinstellato-ms** [reply]
That could be entered as a request, but it's likely functionality that would be required in VS Code first.
